### ISSUE TITLE
Unable to submit (HTTP: 400) - duplicate of previous iteration

### ISSUE THREAD

**Issue by wolf99** [exercism/exercism.io#3359]
I accidentally submitted (on the C track) before I was ready, so I used the web interface to delete my submission.

Now I am ready to submit but running `$ exercism s pangram{.c,.h}` results in:

~~~sh
2017/01/27 11:03:06 unable to submit (HTTP: 400) - duplicate of previous iteration
~~~

Tried altering the files and trying again, but no change. Tried deleting my `pangram` dir and starting from scratch ('fetch' output as below indicated I have not submitted it yet), but still no luck.

~~~sh
$ exercism f c pangram

Not Submitted: 1 problem
c (Pangram) /home/Me/exercism/c/pangram

New:        1 problem
c (Pangram) /home/Me/exercism/c/pangram

unchanged: 0, updated: 0, new: 1
~~~

`debug` output:
~~~sh
$ exercism debug

**** Debug Information ****
Exercism CLI Version: 2.3.0
Exercism CLI Latest Release: 2.3.0
OS/Architecture: linux/amd64
Build OS/Architecture linux/amd64
Home Dir: /home/Me
Config file: /home/Me/.exercism.json
API Key: ********************************
Exercises Directory: /home/Me/exercism
Testing API endpoints reachability
        * GitHub API: https://api.github.com/ [connected] 110.7445ms
        * API: http://exercism.io [connected] 436.4074ms
        * XAPI: http://x.exercism.io [connected] 1.6468597s

If you are having trouble and need to file a GitHub issue (https://github.com/exercism/exercism.io/issues) please include this information (except your API key. Keep that private).
~~~

**Comment by Insti** [comment]
It does not look like you are using the full path to the file that you want to submit.
Which in your case might be: `c/pangram/pangram.c`

You might find you have a `pangram.c` in the Exercism root that is unchanged, but that's not the one you want to be submitting.

**Comment by wolf99** [comment]
@Insti, this is as my bash was already within the pangram src dir.
I tried as you said though, changed to `exercism/` dir and ran `$ exercism s ./c/pangram/src/pangram{.c,.h}`, but no change. Tried again after `touch`-ing each of the files, again no change.

**Comment by Insti** [comment]
Can you submit the solution to a different problem?
What if you replace the contents of `pangram.c` and `.h` with something like 'hello world' (it need not be valid c, just radically different from your previous solution.)
Do you have an Exercism solution link for pangram in your Exercism dashboard?

**Comment by wolf99** [comment]
> Can you submit the solution to a different problem?

I fetched the next exercise, nth-prime, created blank .c and .h files to suit it and tried to submit those - that works fine

>Do you have an Exercism solution link for pangram in your Exercism dashboard?

No, this disappeared when I deleted the exercise using the web interface as mentioned in OP.

>What if you replace the contents of pangram.c and .h with something like 'hello world' (it need not be valid c, just radically different from your previous solution.)

This worked! I changed to a simple hello world function in pangram.c and tried the submit and it went through - then switched back to the correct pangram implementation and that submitted up too. In the web interface, I note that my previous mistaken and deleted iteration is also present (along with the hello world test iteration)
